crates/solver/src/main.rs

```diff
@@ -308,6 +308,14 @@ struct Arguments {
     /// How pending transactions should be fetched.
     #[clap(long, env, arg_enum, default_value = "ignore")]
     pending_transaction_config: PendingTransactionConfig,
+
+    /// Additional configuration for specifying liquidity order owners at the
+    /// driver level.
+    ///
+    /// This has the effect of having these orders only be considered for COWs
+    /// instead of generally trading them against on-chain AMM liquidity.
+    #[clap(long, env, use_value_delimiter = true)]
+    additional_liquidity_order_owners: Vec<H160>,
```

Comment:
If this ends up as part of the driver again let's at least have only a single argument for orderbook and driver and put it into `/shared/arguments`.

Reply:
I had thought of that, but put it here to have different help text as they have slightly different semantics.